web/themes/custom/usagov/templates/views-view-fields--federal-agencies.html.twig

```diff
@@ -37,7 +37,8 @@
 #}
 
 {% set lang = row._entity.langcode.value %}
-{% set UID = 'dr-' ~ fields.title.content|striptags|trim %}
+{% set Title = 'dr-' ~ fields.title.content|striptags|trim %}
```

Comment:
Can we change this to 
{% set UTitle = 'dr-' ~ fields.title.content|striptags|trim %}

Reply:
Button needs to be changed to still use UTitle and set UID in data attribute
```
  <button
      aria-controls="{{- UTitle -}}"
      data-node-id="{{- UID -}}
      aria-expanded="false"
      class="usa-accordion__button"
      >
```

Reply:
Div on line 54 needs to be changed to 
`<div class="usa-accordion__content usa-prose" id="{{- UTitle -}}" data-node-id="{{- UID -}}">
`